src/opm/parser/eclipse/EclipseState/Schedule/Well/WellTestState.cpp

```diff
@@ -58,24 +61,48 @@ namespace Opm {
         return (well_iter != wells.end());
     }
 
+
+    WellTestState::ClosedWell& WellTestState::getWell(const std::string& well_name, WellTestConfig::Reason reason)
+    {
+        const auto well_iter = std::find_if(wells.begin(), wells.end(), [&well_name, &reason](const ClosedWell& well) {
+            return (reason == well.reason && well.name == well_name);
+        });
+
+        assert(well_iter != wells.end());
```

Comment:
Please remove `assert()` - use an exception - or forget it. I don't like how the `assert()` interacts with the compiler flags.